src/web/Server.hpp

```diff
@@ -201,7 +211,7 @@ class Server : public std::enable_shared_from_this<Server<PlainSessionType, SslS
 
     util::Logger log_{"WebServer"};
     std::reference_wrapper<boost::asio::io_context> ioc_;
-    std::optional<std::reference_wrapper<boost::asio::ssl::context>> ctx_;
+    std::optional<boost::asio::ssl::context> ctx_;
```

Comment:
Are we sure this works as expected? i imagine it was previously a reference for a reason?

Reply:
It was a reference to be able to create a context outside of Server. It was created in main and passed by reference. Now Server owns the context.